src/Container/InputPage/TextBox/TextBox.module.css

```
@@ -0,0 +1,11 @@
+.textbox{
+    padding: 0.5rem;
+    outline: 0px;
+    border:0px;
+    letter-spacing: 1px;
+    background-color: var(--neutral-mild);
+}
```

```suggestion
.textbox{
    padding: 0.5rem;
    outline: 0;
    border:0;
    letter-spacing: 1px;
    background-color: var(--neutral-mild);
}
```
Same refactoring here.